descriptor.go

```diff
@@ -93,44 +93,44 @@ type RouterDescriptors struct {
 
 // String implements the String as well as the Object interface.  It returns
 // the descriptor's string representation.
-func (desc *RouterDescriptor) String() string {
+func (rds *RouterDescriptor) String() string {
 
 	return fmt.Sprintf("%s,%s,%s,%d,%d,%s,%d,%s,%s,%s",
-		desc.Fingerprint,
-		desc.Nickname,
-		desc.Address,
-		desc.ORPort,
-		desc.DirPort,
-		desc.Published.Format(time.RFC3339),
-		desc.Uptime,
-		strings.Replace(desc.OperatingSystem, ",", "", -1),
-		strings.Replace(desc.TorVersion, ",", "", -1),
-		strings.Replace(desc.Contact, ",", "", -1))
+		rds.Fingerprint,
+		rds.Nickname,
+		rds.Address,
+		rds.ORPort,
+		rds.DirPort,
+		rds.Published.Format(time.RFC3339),
+		rds.Uptime,
+		strings.Replace(rds.OperatingSystem, ",", "", -1),
+		strings.Replace(rds.TorVersion, ",", "", -1),
+		strings.Replace(rds.Contact, ",", "", -1))
 }
 
 // GetFingerprint implements the Object interface.  It returns the descriptor's
 // fingerprint.
-func (desc *RouterDescriptor) GetFingerprint() Fingerprint {
+func (rds *RouterDescriptor) GetFingerprint() Fingerprint {
 
-	return desc.Fingerprint
+	return rds.Fingerprint
 }
 
 // Length implements the ObjectSet interface.  It returns the length of the
 // router descriptors.
-func (descs *RouterDescriptors) Length() int {
+func (rds *RouterDescriptors) Length() int {
```

Comment:
Note that `RouterDescriptor` and `RouterDescriptors` are two different types. Should we call the arguments `rd` and `rds` to highlight this difference?